recipes/cms/meta.yaml

```diff
@@ -0,0 +1,124 @@
+package:
+  name: cms
+  version: "2.0.0b"
+
+about:
+  home: http://broad-cms.readthedocs.io/en/latest/
+  license: GPLv3
+  summary: "CMS (composite of multiple signals) is a suite of tools for resolving signatures of recent positive selection in population genetic data."
+
+source:
+  fn: cms-2.0.0-beta.tar.gz
+  url: https://github.com/broadinstitute/cms/archive/2.0.0-beta.tar.gz
+  md5: 3e5754bf0a6eeb71cc483dcdae10d8c0
+
+build:
+  number: 0
+  rpaths:
+      - lib/
+      - lib//
+
+requirements:
+  build:
+    #- python
+    - gcc
+    - llvm
+
+  run:
+    - libgcc 
+    - alabaster ==0.7.8
+    - babel ==2.3.3
+    - bcftools ==1.3.1
+    - boltons ==16.4.1
+    - cosi2 ==2.02
+    - curl
+    - htslib ==1.3.1
+    - peewee ==2.8.0
+    - pysam ==0.9.1
+    - python-dateutil ==2.3
+    - samtools ==1.3.1
+    - selscan ==1.1.0b
+    - biopython ==1.67
+    - cloog ==0.18.0
+    - docutils ==0.12
+    - ecdsa ==0.13
+    - gcc ==4.8.5
+    - gmp ==5.1.2
+    - imagesize ==0.7.1
+    - isl ==0.12.2
+    - jinja2 ==2.8
+    - libgcc ==4.8.5
+    - matplotlib ==1.5.1
+    - markupsafe ==0.23
+    - mkl ==11.3.3
+    - mpc ==1.0.1
+    - mpfr ==3.1.2
+    - ncurses ==5.9
+    - openssl ==1.0.2h
+    - pandas ==0.18.1
+    - paramiko ==1.16.0
+    - pip ==8.1.2
+    - pycrypto ==2.6.1
+    - pygments ==2.1.3
+    - python ==3.5.1
```

Comment:
If the python version is omitted, bioconda will build this package for [multiple versions](https://github.com/bioconda/bioconda-recipes/blob/master/scripts/env_matrix.yml#L1-L4) of python. Ideally, `- python` would be the first dependency. It may be necessary to remove the version pinning from some of the other dependencies if builds are not available for all of the python versions in the bioconda build matrix.